docs/Troubleshooting.md

```diff
@@ -1,13 +1,15 @@
 # Troubleshooting
 
-### `this.context.router` is `undefined`
+### `this.props.router` is `undefined`
```

Comment:
I'd drop this entirely; it's not really relevant. Or maybe re-phrase this as "how do I get `props.router`?".